acct-group/headscale/headscale-0-r1.ebuild

```diff
@@ -0,0 +1,8 @@
+# Copyright 2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit acct-group
+
+ACCT_GROUP_ID=600
```

Comment:
I can think of security concern here

but, this should be an open question, do you really want to add separate UID/GID? possible to avoid?

for this, possible two way to go
1) go with separate ID, then need to acquire the id first, may need to send PR to api.git, please check wiki/devmanual

https://devmanual.gentoo.org/ebuild-writing/users-and-groups/index.html
2) is it ok to use less privilege id, I'm not sure if i's ok to use nobody for UID/GID?